environment/prod/deployment/beta/.env.beta.secrets

```diff
@@ -0,0 +1,30 @@
+APP_KEY=ENC[AES256_GCM,data:fWyxvZYL0efOrHXvrG9vess79jHBiwRL5PXBu4DNqqZTU3jySKIgOyz9vUfLGJdiqcBh,iv:oR0MrlXOn3BF3yBftIgHFsbv83j+Bl8GcGEsHLJDpDI=,tag:N8unSLAgjb5gCKI8kq3A3A==,type:str]
+DOCKER_KEATING_BETA_DB_ROOT_PASSWORD=ENC[AES256_GCM,data:DhcuqzMHYQ==,iv:prgSu/J9Q3FVzyo69Gj9CoGH8BHIrPUFwXbwetJSHGc=,tag:q3cSBVn8FMV2DFBMRYG8zA==,type:str]
+DOCKER_KEATING_BETA_DB_DATABASE=ENC[AES256_GCM,data:u2uwoADz8w==,iv:SaYfIc25URQef+KHECmKJj+nKST6F8lnlUPKDVRXgwE=,tag:paQEaoa7GKLSeunQyCfHOw==,type:str]
+DOCKER_KEATING_BETA_DB_USERNAME=ENC[AES256_GCM,data:rwdbtZ/0aA==,iv:U9+Kz+A+gFQG6Yn4eaukqmVcpvYDa1q7JgiqXyFfqpc=,tag:p0oQ24I3fBNqxWKuJUpChQ==,type:str]
+DOCKER_KEATING_BETA_DB_PASSWORD=ENC[AES256_GCM,data:r6kAtRDIruM=,iv:cI3+rda0fIqv9O2W5/kgwtt+jDSn29TYNIhpaT+SqVg=,tag:H1oavvqHWEoLItEif4E0PQ==,type:str]
+DB_CONNECTION=ENC[AES256_GCM,data:pZEeBAQ=,iv:kt9YsArfWKqI0V6IGahSFirC2jMsqLKNb2mdr6zVFlw=,tag:Y1o3FCrq/ZgGldJf6z6UPA==,type:str]
+DB_HOST=ENC[AES256_GCM,data:3qzHHa9DX5RuLK03OeyWvD9bQws0,iv:08d+Xq3LqoR9iTnclUmmpu6R15EWue076mTVlPYhNFY=,tag:UaUscRO2eIAaOwzp+vnnBg==,type:str]
+DB_PORT=ENC[AES256_GCM,data:COVOOw==,iv:r+3V40sN2cN6IkvuF8OjP8Sko+G1YQtzNw3kJrVhpn8=,tag:LdA7Nj3+4IePzZZ7kW10yg==,type:str]
+DB_DATABASE=ENC[AES256_GCM,data:K5vX1dxsPQ==,iv:mUOf/NxbhwBKZX7wzKwWj8AXK3Tu7bNyuFItGkELpuU=,tag:UK1B0PmSArTumucpup7T/w==,type:str]
+DB_USERNAME=ENC[AES256_GCM,data:/guuobLJiw==,iv:SrjmoQIwpzprwoM+3Qing8NhinNKCZOVMdAuHP3iepI=,tag:cmjrd2e6IgZfJ44DNNKHgA==,type:str]
+DB_PASSWORD=ENC[AES256_GCM,data:rQVriWbnkvA=,iv:ryv5mreUMvBfOM8qczEYSJCtUpViOjNXtJpZ8wJWn0U=,tag:T2p9jjRKOXIyUhS1tuTIPA==,type:str]
+DB_ROOT_PASSWORD=ENC[AES256_GCM,data:6uEP3a4g4Q==,iv:W6VDUUYopVbQNhMDdoHjkK3FV/KoS5WbrJcBEJ7ae3g=,tag:WKls4D3L5UkMyIXMFlFe9A==,type:str]
+REDIS_HOST=ENC[AES256_GCM,data:5QzLEB48t0FG+htDhyR873KO,iv:qeZPEEQIDcIF6GyXZ8U5pjQz/M8UfYRbFxP9fiQvPIk=,tag:bBZjL+3k4lk+nCdke6yXpg==,type:str]
```

Comment:
I think there are to much encoded variables. I would like to put here only sensitive data like passwords, app key.

Reply:
I changed it, please check